src/dhali/transaction_utils.py

```diff
@@ -4,6 +4,13 @@
 from fastapi import HTTPException
 from google.cloud import firestore
 
+root_private_collection_name = "payment_channels"
+root_public_collection_name = "public_claim_info"
+estimate_collection_name = "estimate"
+exact_collection_name = "exact"
+request_charge_header_key = "Dhali-Latest-Request-Charge"
+request_total_charge_header_key = "Dhali-Total-Requests-Charge"
```

Comment:
Are we putting these here rather than the config jsons because we don’t have a way to depend on them in Dhali-py?

Reply:
Discussed in person, yes